governance/default_test.go

```diff
@@ -318,13 +320,10 @@ func TestGovernance_ParseVoteValue(t *testing.T) {
 		d := new(GovernanceVote)
 		rlp.DecodeBytes(b, d)
 
-		if d, err = gov.ParseVoteValue(d); err != nil {
-			assert.Equal(t, nil, err)
-		}
-
-		if !reflect.DeepEqual(v, d) {
-			t.Errorf("Parse was not successful! %v %v \n", v, d)
-		}
+		d, err := gov.ParseVoteValue(d)
+		assert.Equal(t, nil, err)
+		assert.Equal(t, reflect.DeepEqual(v, d), true,
```

Comment:
need to try `assert.DeepEqual` instead.

Reply:
Reflected at https://github.com/klaytn/klaytn/pull/1110/commits/70ad97142c82c7962c5150839e6fb074ae5091f3.